feat: figure out binaries upload (#800)
ansys/pyansys-geometry: doc/source/assets.rst

Windows container
^^^^^^^^^^^^^^^^^

.. note::

>    These binaries are only available for download to users with access to
>    https://github.com/ansys/pyansys-geometry-binaries

Reviewer: ```suggestion
   Only users with access to ``https://github.com/ansys/pyansys-geometry-binaries`` can download these binaries.
```
You might want to make the location a link.

* `Latest Geometry service binaries for Windows containers <https://github.com/ansys/pyansys-geometry-binaries>`_
* `Latest Geometry service Dockerfile for Windows containers <https://github.com/ansys/pyansys-geometry/releases/latest/download/windows-dockerfile.zip>`_

Linux container
^^^^^^^^^^^^^^^

.. note::

   These binaries are only available for download to users with access to

>    https://github.com/ansys/pyansys-geometry-binaries

Reviewer: Same change here as earlier.
